src/text_engine/items/ItemBuilder.java

```diff
@@ -1,67 +0,0 @@
-package text_engine.items;
-
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Stack;
-
-import text_engine.effects.Effect;
-import text_engine.items.combinations.Combination;
-import text_engine.items.combinations.Combinations;
-
-public class ItemBuilder {
```

Comment:
I trashed `ItemBuilder` and put the couple of things that were really needed directly on `Item`, set up so that they can be used in a builder-style initialisation pattern.